tests.py

```diff
@@ -61,33 +98,33 @@ class TestPredatorEatsPrey(unittest.TestCase):
     """
 
     def test_predator_eats_new_prey(self):
-        freddie_the_lion = Animal("Freddie", "Lion")
-        bobby_the_sheep = Animal("Bobby", "Sheep")
+        freddie = Lion("Freddie")
```

Comment:
The thing I was talking about with making sure there is no output when running tests, is that you can patch the `print` function so that 1. running tests doesn't give you messy output and 2. you can check that the correct string is (usually) printed.  Have a look at the mock library, particularly mock.patch (https://docs.python.org/3/library/unittest.mock.html).  If you get stuck, speak to Tom who can show you how he did it.